src/expression.d

```diff
@@ -4829,12 +4829,12 @@ public:
      * Convert string contents to a 0 terminated string,
      * allocated by mem.xmalloc().
      */
-    final const(char)* toStringz() const
+    extern (D) final const(char)[] toStringz() const
```

Comment:
Please be really careful when doing this with virtual functions.  The C++ and D sides do not share vtables, so it might not get noticed immediately.